src/OpenTelemetry/Trace/Samplers/AlwaysOffActivitySampler.cs

```diff
@@ -22,7 +22,7 @@ namespace OpenTelemetry.Trace.Samplers
     public sealed class AlwaysOffActivitySampler : ActivitySampler
     {
         /// <inheritdoc />
-        public override string Description { get; } = nameof(AlwaysOffActivitySampler);
+        public override string Description { get; } = "AlwaysOffSampler";
```

Comment:
We'll be renaming ActivitySampler to Sampler anyway, so this change will be non-required. Prefer to avoid changes here to avoid merge conflict with my PR doing the rename.

Reply:
IMO a string constant here is better than `nameof` because it is more reflective of the specific name the spec says `MUST` match. I think having the `nameof` might lead to mistakes where it is changed based on class renames when really it should be detached? LMK Re: The conflict, 🤦 If this gets merged first I can fix that up in your branch, np. I would rather do that than muck with trying to revert the change here.

Reply:
This is fine. I will handle conflict - it not a major issue.